Cargo.toml

```diff
@@ -13,8 +13,9 @@ edition = "2018"
 
 [features]
 default = [ "std" ]
-std = [ "unicode-normalization", "serde/std" ]
+std = [ "alloc", "serde/std", "unicode-normalization/std" ]
```

Comment:
maybe we don't need `alloc` here?
`"unicode-normalization/std"` will auto pull `unicode-normalization`

Reply:
Yes but we do not need the `unicode-normalization` "std" feature so why enable it? Also "unicode-normalization" is enabled already in "alloc".

Reply:
On further consideration, why are we even enabling "unicode-normalization" in "alloc"? Seems that `cargo check --no-default-features --features=unicode-normalization` builds with this feature set
```rust
default = [ "std" ]
std = [ "alloc", "serde/std" ]
alloc = []
```

(excl. `rand` stuff for brevity)

Reply:
Wow, these features are a mess - "serde/std" is wrong too.

Reply:
> On further consideration, why are we even enabling "unicode-normalization" in "alloc"? Seems that cargo check --no-default-features --features=unicode-normalization builds with this feature set

I was considering this, we can do it.

But `unicode-normalization` is just part of _normal_ implementation of bip39. It was unconditionally enabled in `std` and it was not exposed to external world. So I decided to add it also in `alloc` as a default feature.